src/sentry/tagstore/snuba/backend.py

```diff
@@ -1489,3 +1582,7 @@ def get_group_event_filter(self, project_id, group_id, environment_ids, tags, st
             return None
 
         return {"event_id__in": event_id_set}
+        if not event_id_set:
```

Comment:
Why did you add this?

Reply:
Must be something duplicated on a merge, wasn't intentional